src/main/shell-session.ts

```diff
@@ -108,7 +108,7 @@ export class ShellSession extends EventEmitter {
 
     if(isWindows) {
       env["SystemRoot"] = process.env.SystemRoot;
-      env["PTYSHELL"] = "powershell.exe";
+      env["PTYSHELL"] = process.env.SHELL || "powershell.exe";
```

Comment:
Shouldn't we only optionally override `PTYSHELL`?
```suggestion
      env["PTYSHELL"] ||= process.env.SHELL || "powershell.exe";
```

Reply:
Maybe, I tried to keep this in line with existing logic. We could open an issue about this if needed.